qcodes/instrument_drivers/AlazarTech/ATS9440.py

```diff
@@ -0,0 +1,311 @@
+                                                     validators.Enum('UNDEFINED')),
+                           initial_value='UNDEFINED')
+        self.add_parameter(name='sample_rate',
+                           get_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='Internal Sample Rate',
+                           unit='S/s',
+                           initial_value=100000000,
+                           val_mapping={1_000: 1,
+                                        2_000: 2,
+                                        5_000: 4,
+                                       10_000: 8,
+                                       20_000: 10,
+                                       50_000: 12,
+                                      100_000: 14,
+                                      200_000: 16,
+                                      500_000: 18,
+                                    1_000_000: 20,
+                                    2_000_000: 24,
+                                    5_000_000: 26,
+                                   10_000_000: 28,
+                                   20_000_000: 30,
+                                   50_000_000: 34,
+                                  100_000_000: 36,
+                                  125_000_000: 38,
+                             'EXTERNAL_CLOCK': 64,
+                                  'UNDEFINED': 'UNDEFINED'})
+        self.add_parameter(name='clock_edge',
+                           get_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='Clock Edge',
+                           unit=None,
+                           initial_value='CLOCK_EDGE_RISING',
+                           val_mapping={'CLOCK_EDGE_RISING': 0,
+                                        'CLOCK_EDGE_FALLING': 1})
+        self.add_parameter(name='decimation',
+                           get_cmd=None,
+                           parameter_class=TraceParameter,
+                           label='Decimation',
+                           unit=None,
+                           initial_value=1,
+                           vals=validators.Ints(1, 100000))
+        for i in ['1', '2']:
+            self.add_parameter(name='coupling' + i,
+                               get_cmd=None,
+                               parameter_class=TraceParameter,
+                               label='Coupling channel ' + i,
+                               unit=None,
+                               initial_value='DC',
+                               val_mapping={'AC': 1, 'DC': 2})
+            self.add_parameter(name='channel_range' + i,
+                               get_cmd=None,
+                               parameter_class=TraceParameter,
+                               label='Range channel ' + i,
+                               unit='V',
+                               initial_value=0.4,
```

Comment:
initial_value=0.1 (it maybe is a good idea to set the initial value to the minimum)

Reply:
changed